packages/core/src/contract-info.ts

```diff
@@ -34,84 +36,105 @@ import {
-    ],
-  },
-  {
-    artifact: ProxyArtifact,
-    expectedAddress: REFERENCE_PROXY_ADDRESS,
-    constructorArgs: [getChugSplashRegistryAddress()],
-  },
-  {
-    artifact: ForwarderArtifact,
-    expectedAddress: FORWARDER_ADDRESS,
-    constructorArgs: [],
-  },
-]
+}> => {
+  const lzEndpointAddress =
+    LAYERZERO_ENDPOINT_ADDRESSES[chainId]?.address ??
+    getMockEndPointAddress(chainId)
+  return [
+    {
+      artifact: ChugSplashRegistryArtifact,
+      expectedAddress: getChugSplashRegistryAddress(),
+      constructorArgs: getRegistryConstructorValues(),
+    },
+    {
+      artifact: ChugSplashManagerArtifact,
+      expectedAddress: getChugSplashManagerV1Address(),
+      constructorArgs: getManagerConstructorValues(),
+    },
+    {
+      artifact: DefaultAdapterArtifact,
+      expectedAddress: DEFAULT_ADAPTER_ADDRESS,
+      constructorArgs: [DEFAULT_UPDATER_ADDRESS],
+    },
+    {
+      artifact: OZUUPSOwnableAdapterArtifact,
+      expectedAddress: OZ_UUPS_OWNABLE_ADAPTER_ADDRESS,
+      constructorArgs: [OZ_UUPS_UPDATER_ADDRESS],
+    },
+    {
+      artifact: OZUUPSAccessControlAdapterArtifact,
+      expectedAddress: OZ_UUPS_ACCESS_CONTROL_ADAPTER_ADDRESS,
+      constructorArgs: [OZ_UUPS_UPDATER_ADDRESS],
+    },
+    {
+      artifact: OZTransparentAdapterArtifact,
+      expectedAddress: OZ_TRANSPARENT_ADAPTER_ADDRESS,
+      constructorArgs: [DEFAULT_UPDATER_ADDRESS],
+    },
+    {
+      artifact: DefaultUpdaterArtifact,
+      expectedAddress: DEFAULT_UPDATER_ADDRESS,
+      constructorArgs: [],
+    },
+    {
+      artifact: OZUUPSUpdaterArtifact,
+      expectedAddress: OZ_UUPS_UPDATER_ADDRESS,
+      constructorArgs: [],
+    },
+    {
+      artifact: DefaultCreate3Artifact,
+      expectedAddress: DEFAULT_CREATE3_ADDRESS,
+      constructorArgs: [],
+    },
+    {
+      artifact: DefaultGasPriceCalculatorArtifact,
+      expectedAddress: DEFAULT_GAS_PRICE_CALCULATOR_ADDRESS,
+      constructorArgs: [],
+    },
+    {
+      artifact: ManagedServiceArtifact,
+      expectedAddress: MANAGED_SERVICE_ADDRESS,
+      constructorArgs: [getOwnerAddress()],
+    },
+    {
+      artifact: ChugSplashManagerProxyArtifact,
+      expectedAddress: REFERENCE_CHUGSPLASH_MANAGER_PROXY_ADDRESS,
+      constructorArgs: [
+        getChugSplashRegistryAddress(),
+        getChugSplashRegistryAddress(),
+      ],
+    },
+    {
+      artifact: ProxyArtifact,
+      expectedAddress: REFERENCE_PROXY_ADDRESS,
+      constructorArgs: [getChugSplashRegistryAddress()],
+    },
+    {
+      artifact: ForwarderArtifact,
+      expectedAddress: FORWARDER_ADDRESS,
+      constructorArgs: [],
+    },
+    {
+      artifact: FunderArtifact,
+      expectedAddress: getFunderAddress(lzEndpointAddress),
+      constructorArgs: [lzEndpointAddress],
+    },
+    {
+      artifact: LZReceiverArtifact,
+      expectedAddress: getLZReceiverAddress(lzEndpointAddress),
+      constructorArgs: [lzEndpointAddress],
+    },
```

Comment:
LayerZero endpoint contracts aren't consistent across networks, so we have to fetch the correct addresses based on the chain id.